spec/lib/jobs/isc-dhcp-poller-job-spec.js

```diff
@@ -7,12 +7,13 @@ describe('ISC DHCP Poller Job', function () {
     var base = require('./base-spec');
     var uuid;
 
-    /// create a future end date
-    var todayDate = new Date();
-    // add 30 seconds
-    var newdate = new Date(todayDate.getTime() + (10000));
+    // create a future end date
+    // consider different timezone and the daylight saving, add 2 days bases on now will always
+    // generate a future date if discard timezone while parsing.
+    var futureDate = new Date();
+    futureDate.setDate(futureDate.getDate() + 2); //add 2 days to current
```

Comment:
@tldavies could you please review this change, as I see you are the author who created the first line code. 

This is to fix the unit-test failure for isc-dhcp-lease-poller, I guess this problem is caused by the daylight saving change, because we had not found this issue before. Previous calculation for future only adds extra 10000ms, but the below line `newdate = newdate.split('.')[0].replace('T', ' ').replace(/-/g, '/');` will remove some critical timing info, so when it is eventually parsed, the time may not be in the future.

My fix is to add extra 2 days into the calculation, so regardless the daylight saving and timezone, the new date will always be in the future.

I admit the best solution is to clarify those date format difference (UTC/ISO/Local, .etc) and understand all necessary Javascript Date function, but that's will take relative more time for me and I think for unit-testing, simply add 2 days to generate future days is enough.